### PR TITLE
doc: fast_pair: known_issues: Add rebond fail known issue

### PR DESCRIPTION
**Pull request by alstrzebonski**
Informs about known issue with rebonding.

Jira: NCSDK-23682

### CODE REVIEW

File: doc/nrf/releases_and_maturity/known_issues.rst [hunk 1]
```diff
@@ -2271,6 +2271,14 @@ NCSDK-23315: The :ref:`bt_le_adv_prov_readme` has an incorrect range and default
 
   **Workaround:** Manually cherry-pick and apply the commit with the fix from the ``main`` branch (commit hash: ``a8b668e82837295962348e9e681125c2ae11bb4e``).
 
+.. rst-class:: v2-5-0 v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0
+
+NCSDK-23682: Fast Pair Seeker may be unable to bond again after it loses bond multiple times when it uses RPA address and the :kconfig:option:`CONFIG_BT_SETTINGS_CCC_LAZY_LOADING` Kconfig option is disabled on Fast Pair Provider side.
```
Comment (kapi-no) [inline doc/nrf/releases_and_maturity/known_issues.rst:2276]:
```suggestion
NCSDK-23682: If the :kconfig:option:`CONFIG_BT_SETTINGS_CCC_LAZY_LOADING` Kconfig option is disabled on the Fast Pair Provider side, the Fast Pair Seeker that uses the RPA address to connect with the Provider may be unable to bond again after losing the bonding information multiple times.
```

File: doc/nrf/releases_and_maturity/known_issues.rst [hunk 2]
```diff
@@ -2271,6 +2271,14 @@ NCSDK-23315: The :ref:`bt_le_adv_prov_readme` has an incorrect range and default
...
+
+The issue is because of a `Zephyr issue #64042`_.
```
Comment (kapi-no) [inline doc/nrf/releases_and_maturity/known_issues.rst:2278]:
```suggestion
The issue is a consequence of `Zephyr issue #64042`_.
```

**Comment by peknis** [comment]
Thanks @alstrzebonski. I believe it would be better to add this item directly to @divipillai's Known issues PR https://github.com/nrfconnect/sdk-nrf/pull/12657. What do you think Divya? If agreed, you could then close this one.

**Comment by divipillai** [comment]
> Thanks @alstrzebonski. I believe it would be better to add this item directly to @divipillai's Known issues PR #12657. What do you think Divya? If agreed, you could then close this one.

yeah, please add this to PR 12657.

**Comment by alstrzebonski** [comment]
> > Thanks @alstrzebonski. I believe it would be better to add this item directly to @divipillai's Known issues PR #12657. What do you think Divya? If agreed, you could then close this one.
> 
> yeah, please add this to PR 12657.

Done

**Comment by alstrzebonski** [comment]
Will be done here: https://github.com/nrfconnect/sdk-nrf/pull/12773

**Pull request closed without merging**